src/settings.c

```diff
@@ -41,6 +41,14 @@ static void on_sidebar_pos_left_changed(GSettings *settings, gchar *key, gpointe
 }
 
 
+static void on_editor_font_changed(GSettings *settings, gchar *key, gpointer user_data)
+{
```

Comment:
I'm not sure if such genericity is feasible at this point, though I could be mistaken.